### ISSUE TITLE
Crash when starting application

### ISSUE THREAD

**Issue by Tunous** [Tunous/Dawn#189]
**Describe the bug**
Application doesn’t launch

**To Reproduce**
1. Start application

**Expected behavior**
Application launches

**Smartphone (please complete the following information):**
- Model Name: P8 Lite
- RAM: 2048 MB
- Manufacturer: Huawei
- OpenGL ES Version: 2.0
- Android Version: Android 5.0
- Native platform: armeabi-v7a
- Locale: en_US
- CPU Make: HiSilicon

**Additional context**

<details>
<summary>Stack-trace</summary>

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{me.thanel.dank/me.saket.dank.ui.subreddit.SubredditActivity}: android.view.InflateException: Binary XML file line #11: Error inflating class me.saket.dank.widgets.InboxUI.InboxRecyclerView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2406)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)

Caused by: android.view.InflateException: Binary XML file line #11: Error inflating class me.saket.dank.widgets.InboxUI.InboxRecyclerView
        at android.view.LayoutInflater.createView(LayoutInflater.java:652)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.createCustomViewInternal(-ViewPumpLayoutInflater.kt:220)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.access$createCustomViewInternal(-ViewPumpLayoutInflater.kt:19)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(-ViewPumpLayoutInflater.kt:421)
        at io.github.inflationx.viewpump.internal.-FallbackViewCreationInterceptor.intercept(-FallbackViewCreationInterceptor.kt:14)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at me.saket.dank.ui.preferences.TypefaceInflationInterceptor.intercept(TypefaceInflationInterceptor.java:31)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.kt:36)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(-ViewPumpLayoutInflater.kt:399)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:180)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:752)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:57)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:48)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:368)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at me.saket.dank.ui.subreddit.SubredditActivity.onCreate(SubredditActivity.java:182)
        at android.app.Activity.performCreate(Activity.java:6013)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2359)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)

Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Constructor.newInstance(Constructor.java:-2)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
        at android.view.LayoutInflater.createView(LayoutInflater.java:626)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.createCustomViewInternal(-ViewPumpLayoutInflater.kt:220)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.access$createCustomViewInternal(-ViewPumpLayoutInflater.kt:19)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(-ViewPumpLayoutInflater.kt:421)
        at io.github.inflationx.viewpump.internal.-FallbackViewCreationInterceptor.intercept(-FallbackViewCreationInterceptor.kt:14)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at me.saket.dank.ui.preferences.TypefaceInflationInterceptor.intercept(TypefaceInflationInterceptor.java:31)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.kt:36)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(-ViewPumpLayoutInflater.kt:399)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:180)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:752)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:57)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:48)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:368)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at me.saket.dank.ui.subreddit.SubredditActivity.onCreate(SubredditActivity.java:182)
        at android.app.Activity.performCreate(Activity.java:6013)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2359)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)

Caused by: android.content.res.Resources$NotFoundException: File res/drawable/scrollbar_handle_material.xml from drawable resource ID #0x10805d5
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2450)
        at android.content.res.Resources.loadDrawable(Resources.java:2350)
        at android.content.res.HwResources.loadDrawable(HwResources.java:589)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:788)
        at android.view.View.initializeScrollbarsInternal(View.java:4533)
        at android.view.View.<init>(View.java:4160)
        at android.view.ViewGroup.<init>(ViewGroup.java:491)
        at android.view.ViewGroup.<init>(ViewGroup.java:487)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:651)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:647)
        at me.saket.dank.widgets.InboxUI.InboxRecyclerView.<init>(InboxRecyclerView.java:48)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:-2)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
        at android.view.LayoutInflater.createView(LayoutInflater.java:626)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.createCustomViewInternal(-ViewPumpLayoutInflater.kt:220)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.access$createCustomViewInternal(-ViewPumpLayoutInflater.kt:19)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(-ViewPumpLayoutInflater.kt:421)
        at io.github.inflationx.viewpump.internal.-FallbackViewCreationInterceptor.intercept(-FallbackViewCreationInterceptor.kt:14)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at me.saket.dank.ui.preferences.TypefaceInflationInterceptor.intercept(TypefaceInflationInterceptor.java:31)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.kt:36)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(-ViewPumpLayoutInflater.kt:399)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:180)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:752)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:57)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:48)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:368)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at me.saket.dank.ui.subreddit.SubredditActivity.onCreate(SubredditActivity.java:182)
        at android.app.Activity.performCreate(Activity.java:6013)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2359)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)

Caused by: android.content.res.Resources$NotFoundException: File res/color/text_color_secondary.xml from color state list resource ID #0x7f0601c8
        at android.content.res.Resources.loadColorStateList(Resources.java:2565)
        at android.content.res.HwResources.loadColorStateList(HwResources.java:717)
        at android.content.res.TypedArray.getColorStateList(TypedArray.java:448)
        at android.graphics.drawable.NinePatchDrawable.updateStateFromTypedArray(NinePatchDrawable.java:465)
        at android.graphics.drawable.NinePatchDrawable.inflate(NinePatchDrawable.java:401)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1095)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1017)
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2440)
        at android.content.res.Resources.loadDrawable(Resources.java:2350)
        at android.content.res.HwResources.loadDrawable(HwResources.java:589)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:788)
        at android.view.View.initializeScrollbarsInternal(View.java:4533)
        at android.view.View.<init>(View.java:4160)
        at android.view.ViewGroup.<init>(ViewGroup.java:491)
        at android.view.ViewGroup.<init>(ViewGroup.java:487)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:651)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:647)
        at me.saket.dank.widgets.InboxUI.InboxRecyclerView.<init>(InboxRecyclerView.java:48)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:-2)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
        at android.view.LayoutInflater.createView(LayoutInflater.java:626)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.createCustomViewInternal(-ViewPumpLayoutInflater.kt:220)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.access$createCustomViewInternal(-ViewPumpLayoutInflater.kt:19)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(-ViewPumpLayoutInflater.kt:421)
        at io.github.inflationx.viewpump.internal.-FallbackViewCreationInterceptor.intercept(-FallbackViewCreationInterceptor.kt:14)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at me.saket.dank.ui.preferences.TypefaceInflationInterceptor.intercept(TypefaceInflationInterceptor.java:31)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.kt:36)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(-ViewPumpLayoutInflater.kt:399)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:180)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:752)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:57)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:48)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:368)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at me.saket.dank.ui.subreddit.SubredditActivity.onCreate(SubredditActivity.java:182)
        at android.app.Activity.performCreate(Activity.java:6013)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2359)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)

Caused by: java.lang.RuntimeException: not a float!
        at android.content.res.XmlBlock$Parser.getAttributeFloatValue(XmlBlock.java:422)
        at android.content.res.ColorStateList.inflate(ColorStateList.java:212)
        at android.content.res.ColorStateList.createFromXmlInner(ColorStateList.java:152)
        at android.content.res.ColorStateList.createFromXml(ColorStateList.java:131)
        at android.content.res.Resources.loadColorStateList(Resources.java:2561)
        at android.content.res.HwResources.loadColorStateList(HwResources.java:717)
        at android.content.res.TypedArray.getColorStateList(TypedArray.java:448)
        at android.graphics.drawable.NinePatchDrawable.updateStateFromTypedArray(NinePatchDrawable.java:465)
        at android.graphics.drawable.NinePatchDrawable.inflate(NinePatchDrawable.java:401)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1095)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1017)
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2440)
        at android.content.res.Resources.loadDrawable(Resources.java:2350)
        at android.content.res.HwResources.loadDrawable(HwResources.java:589)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:788)
        at android.view.View.initializeScrollbarsInternal(View.java:4533)
        at android.view.View.<init>(View.java:4160)
        at android.view.ViewGroup.<init>(ViewGroup.java:491)
        at android.view.ViewGroup.<init>(ViewGroup.java:487)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:651)
        at androidx.recyclerview.widget.RecyclerView.<init>(RecyclerView.java:647)
        at me.saket.dank.widgets.InboxUI.InboxRecyclerView.<init>(InboxRecyclerView.java:48)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:-2)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
        at android.view.LayoutInflater.createView(LayoutInflater.java:626)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.createCustomViewInternal(-ViewPumpLayoutInflater.kt:220)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.access$createCustomViewInternal(-ViewPumpLayoutInflater.kt:19)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(-ViewPumpLayoutInflater.kt:421)
        at io.github.inflationx.viewpump.internal.-FallbackViewCreationInterceptor.intercept(-FallbackViewCreationInterceptor.kt:14)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at me.saket.dank.ui.preferences.TypefaceInflationInterceptor.intercept(TypefaceInflationInterceptor.java:31)
        at io.github.inflationx.viewpump.internal.-InterceptorChain.proceed(-InterceptorChain.kt:30)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.kt:36)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(-ViewPumpLayoutInflater.kt:399)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:180)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:752)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:825)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:523)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:57)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:425)
        at io.github.inflationx.viewpump.internal.-ViewPumpLayoutInflater.inflate(-ViewPumpLayoutInflater.kt:48)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:368)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at me.saket.dank.ui.subreddit.SubredditActivity.onCreate(SubredditActivity.java:182)
        at android.app.Activity.performCreate(Activity.java:6013)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2359)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
        at android.app.ActivityThread.access$1200(ActivityThread.java:152)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5538)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

</details>

**Comment by saket** [comment]
Looks like Huawei is missing with system resources.

**Comment by Tunous** [comment]
Should be resolved by #318 as this occurred with Android 5.0
